### PR TITLE
Update build and processes for translation catalogs

### PR DESCRIPTION
**Pull request by coldacid**
Build catalog files from translation files and integrate them into the release process. This completes items 1-3 in #115.

### CODE REVIEW

**Comment by walkero-gr** [comment]
@coldacid Hello there. Can you please check these conflicts?

**Comment by coldacid** [comment]
Yeah I'll fix things up during my lunch.

File: Makefile.linux [hunk 1]
```diff
@@ -237,6 +244,10 @@ release:
 	cp required_files/iGame.info iGame_rel/iGame-$(DATE)/
 	cp required_files/igame_drawer_3.0.info iGame_rel/iGame.info
 	cp guide/* iGame_rel/iGame-$(DATE)/
+	mkdir iGame_rel/iGame-$(DATE)/catalogs
+	cp catalogs/iGame.cd iGame_rel/iGame-$(DATE)/catalogs/
+	cd iGame_rel/iGame-$(DATE) && mkdir $(catalog_dirs)
```
[the remark this answers is not in the document]
Comment (coldacid) [reply Makefile.linux:249]: It's defined on line 80 of the changed version of the file:

```
catalog_files := $(patsubst %/iGame.ct,%/iGame.catalog,$(wildcard catalogs/*/iGame.ct))
```

**Comment by coldacid** [comment]
@walkero-gr rebased and conflicts eliminated.

**Pull request merged**